docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -380,14 +380,16 @@ There is no limit on field descriptions for either key or message.
 Confluent table description supplier
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Confluent table description supplier uses the `Confluent Schema Registry
-<https://docs.confluent.io/1.0/schema-registry/docs/intro.html>`_ to discover
-table definitions. Set ``kafka.table-description-supplier`` to ``CONFLUENT`` to
-use it and configure the additional properties in the following table.
-
-The benefits of using the schema registry is that new tables can be defined
-without a cluster restart, schema updates are detected and there is no need to
-define tables manually.
+The Kafka connector only uses the `Confluent Schema Registry
+<https://docs.confluent.io/1.0/schema-registry/docs/intro.html>`_ to
+discover table definitions. The benefits of using the schema registry are:
+
+* New tables can be defined without a cluster restart.
+* Schema updates are detected.
```

Comment:
@hashhar ... are detected how? Automatically?

Reply:
Automatically. It's a black from our perspective and user's too. The Kafka library Confluent provides handles all of this.